src/app/routes/app-routes.tsx

```diff
@@ -86,6 +87,88 @@ function useAppRoutes() {
     </Route>
   );
 
+  const legacyRequestRoutes = (
+    <>
+      <Route
+        path={RouteUrls.TransactionRequest}
+        element={
+          <AccountGate>
+            <Suspense fallback={<LoadingSpinner height="600px" />}>
+              <TransactionRequest />
+            </Suspense>
+          </AccountGate>
+        }
+      >
+        {ledgerStacksTxSigningRoutes}
+        <Route path={RouteUrls.EditNonce} element={<EditNonceDrawer />} />
+        <Route path={RouteUrls.TransactionBroadcastError} element={<BroadcastErrorDrawer />} />
+      </Route>
+      <Route
+        path={RouteUrls.SignatureRequest}
+        element={
+          <AccountGate>
+            <Suspense fallback={<LoadingSpinner height="600px" />}>
+              <StacksMessageSigningRequest />
+            </Suspense>
+          </AccountGate>
+        }
+      >
+        {ledgerStacksMessageSigningRoutes}
+      </Route>
+      <Route
+        path={RouteUrls.ProfileUpdateRequest}
+        element={
+          <AccountGate>
+            <Suspense fallback={<LoadingSpinner height="600px" />}>
+              <ProfileUpdateRequest />
+            </Suspense>
+          </AccountGate>
+        }
+      />
+      <Route
+        path={RouteUrls.PsbtRequest}
+        element={
+          <AccountGate>
```

Comment:
Maybe we can extend `AccountGate` to accept a prop for a suspense wrapper? 

Then we won't need to repeat the `<Suspense fallback={<LoadingSpinner height="600px" />}>` so often

Reply:
Good suggestion, I'll try to take on tackling that here.

Reply:
@pete-watters can you make an issue for this? This refactor is quite large already and this code was just moved, so I'm hoping to avoid taking this on here. You can use the label on the issue `code health`.

Reply:
Sure, no problem. I agree his is already big enough 👍 #4051